test/python_rules/worker_test.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+
+"""
```

Comment:
Docstring should be at the top of the file, no?

Reply:
ooopssy!